scripts/library.py

```diff
@@ -103,6 +103,7 @@ def weighted_avg(freqs):
   
   for val in freqs:
     if val < -75: # filter out quiet/ambient noise
+      val *= 0.8
```

Comment:
Is this line just changing any data values below -75 to be 80% of their normal magnitude?